packages/extension/src/newtab/DndModal.tsx

```diff
@@ -1,95 +1,104 @@
+  const [dndTime, setDndTime] = useState<TimeFormat>('HALF_HOUR');
 
   const handleSubmit = async (e: React.MouseEvent<Element, MouseEvent>) => {
-    const expiration =
-      dndTime === TimeFormat.CUSTOM
-        ? getExpiration(customTimeOptions[customTimeIndex], customNumber)
-        : getTimeFormatExpiration(dndTime);
-
+    const option = dndOption[dndTime];
+    const customTime = customTimeOptions[customTimeIndex];
+    const expiration = option.getExpiration(customTime, customNumber);
     const fallback = link || getDefaultLink();
 
     await setDndSettings({ expiration, link: fallback });
 
     onRequestClose(e);
   };
 
-  const renderForm = () => {
-    if (isActive) return <br />;
+  const turnedOnContent = (
+    <div className="grid grid-cols-2 gap-6 justify-center my-6">
+      <button
+        type="submit"
+        onClick={onRequestClose}
+        className="flex justify-center items-center w-28 h-12 rounded-xl border border-white text-base text-white font-bold"
+      >
+        Cancel
+      </button>
+      <button
+        type="submit"
+        onClick={() => setDndSettings(null)}
+        className="flex justify-center items-center w-28 h-12 rounded-xl text-base text-white font-bold bg-onion-40"
+      >
+        Turn Off
+      </button>
+    </div>
+  );
 
-    return (
-      <div className={styles.content}>
-        <TextField
-          className={styles.url}
-          inputId="defaultURL"
-          label="Default URL (optional)"
-          valueChanged={(text) => setLink(text)}
-        />
-        <Radio
-          className="mt-8 "
-          name="timeOff"
-          value={dndTime}
-          options={timeFormatOptions}
-          onChange={(value: TimeFormat) => setDndTime(value)}
-        />
-        {dndTime !== TimeFormat.CUSTOM ? null : (
-          <div className="grid grid-cols-2 gap-4 mt-4">
-            <TextField
-              className={styles.custom}
-              inputId="defaultURL"
-              label="Number"
-              type="number"
-              valueChanged={(number) => setCustomNumber(parseInt(number))}
-            />
-            <Dropdown
-              className={styles.custom}
-              options={customTimeOptions}
-              selectedIndex={customTimeIndex}
-              onChange={(_, index) => setCustomTimeIndex(index)}
-            />
-          </div>
-        )}
-      </div>
-    );
+  const turnedOffContent = (
+    <div className="w-full mt-2 py-4 px-10">
+      <TextField
+        inputId="defaultURL"
+        label="Default URL (optional)"
+        valueChanged={(text) => setLink(text)}
+      />
+      <Radio
+        className="mt-8 "
+        name="timeOff"
+        value={dndTime}
+        options={timeFormatOptions}
+        onChange={(value: TimeFormat) => setDndTime(value)}
+      />
+      {dndTime !== 'CUSTOM' ? null : (
+        <div className="grid grid-cols-2 gap-4 mt-4">
+          <TextField
+            className="w-40 appearance-none"
```

Comment:
```suggestion
            className="w-40"
```

The `appearance-none` is not necessary IMO. Please validate it.

Reply:
Ah, true. It seems `appearance-none` doesn't work anymore, the input type of number results to the image below:

![Screen Shot 2021-08-01 at 7 58 54 PM](https://user-images.githubusercontent.com/13744167/127777495-c4b2bf0d-0af0-4132-827e-b32e7325f6e5.png)

We can add an extended config like the link below as tailwind doesn't support hiding it out-of-the-box yet.

https://github.com/tailwindlabs/tailwindcss/discussions/3275

For the meantime, I'll omit the `appearance-none`

Reply:
If omitting it makes some glitches, we can add it to the text field component itself. This way it will fix it throughout the application. What do you think?

Reply:
That's so much better. I'll commit that in a sec.